erlang/api_cowboy/apps/api_cowboy/src/api_cowboy_app.erl

```diff
@@ -0,0 +1,17 @@
+-module(api_cowboy_app).
+-behaviour(application).
+
+-export([start/2, stop/1]).
+
+start(_StartType, _StartArgs) ->
+    Dispatch = cowboy_router:compile([
+        {'_', [
+            {"/", api_cowboy_handler, []},
+            {"/api/v1/items", api_cowboy_handler, []},
+            {"/api/v1/items/:name", api_cowboy_handler, []}
```

Comment:
You could save this list in a variable named Routes or something like that for better readability

```erlang
Routes = [
  {"/", api_cowboy_handler, []}
],
 Dispatch = cowboy_router:compile([{'_', Routes}]).
```